Alignment/OfflineValidation/plugins/PrimaryVertexValidation.h

```diff
@@ -1,72 +1,73 @@
 #ifndef PrimaryVertexValidation_h
 #define PrimaryVertexValidation_h
 
-#include "FWCore/Framework/interface/Frameworkfwd.h"
-#include "FWCore/Framework/interface/EDAnalyzer.h"
-#include "FWCore/Framework/interface/Event.h"
-#include "FWCore/Framework/interface/ESHandle.h"
-#include "FWCore/Framework/interface/EventSetup.h"
-#include "FWCore/Framework/interface/MakerMacros.h"
-#include "FWCore/ParameterSet/interface/ParameterSet.h"
+// system include files
+#include <iostream>
+#include <memory>
```

Comment:
@mmusich I think, iostream and memory are not needed for your plugin, if I am not mistaken